contracts/libraries/PoolTicksHelper.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+import '@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol';
+
+pragma solidity >=0.6.0;
+
+
+library PoolTicksHelper {
```

Comment:
helper is an overloaded/unhelpful word. `Counter` or no suffix preferred, also some natspec on the library maybe?